console/billing/how-to/change-payment-method.mdx

```diff
@@ -0,0 +1,33 @@
+---
+meta:
+  title: How to change your payment method
+  description: This page explains how to change your payment method
+content:
+  h1: How to change your payment method
+  paragraph: This page explains how to change your payment method
+tags: billing change payment method
+dates:
+  validation: 2024-02-15
+  posted: 2024-02-15
+categories:
+  - billing
+---
+
+If you have registered both a **credit card** and a **SEPA mandate**, you can choose the default payment method you wish to use for your Scaleway account.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+</Message>
+
+1. Click **Billing** in the Organization drop-down menu to go to the Billing Overview page.
+2. Click **Payment and billing**. 
+3. Scroll down to find **Payment Methods**.
+2. Select the payment method you wish to use as default.
+
+    Your new billing address is now used for your invoices.
```

Comment:
Isn't this doc to update the payment method ?

Reply:
> Isn't this doc to update the payment method ?

In the present doc, it says change. I wanted to keep change and not update before they are not adding new information (Ex: credit card details) but instead changing from one default payment method to another.

Reply:
I think Samy means that in the sentence it says "billing address"

Reply:
Ah okay :)).. I will remove the sentence !